src/uwtools/tests/test_cli.py

```diff
@@ -30,6 +30,14 @@ def test__abort(capsys):
     assert msg in capsys.readouterr().err
 
 
+def test__abort_debug_logs_stacktrace(capsys):
+    msg = "Traceback (most recent call last)"
+    with patch.object(sys, "argv", STR.debug):
+        with raises(SystemExit):
+            cli._abort(msg)
+    assert msg in capsys.readouterr().err
```

Comment:
I'm curious if this works out as expected. I had expected that `logging.exception` would only work in a `try:` block.

Actually, the docs say "This method should only be called from an exception handler." Can we treat `_abort` as an exception handler in this case?

Reply:
I might have gotten carried away with `sys` use this time - I'll walk through the breakpoint to make sure its actually doing what I want

Reply:
Good catch, this test was self-defining. With this edit pushed, its not working as expected, so I'll see if the `try`: block works now

Reply:
OK, moved to the try block, but now I'm having issues capturing the logged trace - pushing for feedback